action.yml

```diff
@@ -62,17 +74,40 @@ runs:
         "
 
         echo "Detected bad alt text: ${flag}"
+        echo ${{github.event_name}}
         echo "Event type: $type"
-
-        if [[ $flag = true ]]; then
-          if [[ $type = pr_comment ]] || [[ $type = pr_description ]]; then
-            gh pr comment $issue_url --body "$message"
-          elif [[ $type = issue_comment ]] || [[ $type = issue_description ]]; then
-            gh issue comment $issue_url --body "$message"
-          elif [[ $type = discussion_description ]]; then
-            addDiscussionComment $discussion_node_id "$message"
-          elif [[ $type = discussion_comment ]]; then
-            addDiscussionComment $discussion_node_id "$message" $reply_to_id 
+        echo ${{ github.event.action }}
```

Comment:
Are these logs intentionally left in? If so, would we want to prefix it with additional context?

Reply:
They were for testing I can remove them!